conda/recipes/dask-cudf/meta.yaml

```diff
@@ -19,11 +19,17 @@ build:
   script_env:
     - VERSION_SUFFIX
     - PARALLEL_LEVEL
-    - CC
-    - CXX
-    - CUDAHOSTCXX
+  ignore_run_exports_from:
+    - nvcc_linux-64 # [linux64]
+    - nvcc_linux-aarch64 # [aarch64]
 
 requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }} {{ cuda_version }}
```

Comment:
It might be possible to remove CUDA compiler requirement -- I don't think there are any CUDA-compiled pieces in dask-cudf. The only catch is that `setup.py` checks the installed CUDA toolkit to determine which version of cupy to use from pip! That doesn't apply in the conda builds, though, because matching cupy and CUDA toolkit versions are handled by conda. 😕 https://github.com/rapidsai/cudf/blob/e58d0490472467c724cd9d9e5ec1c6f6efa4beae/python/dask_cudf/setup.py#L68-L70

See also: https://pypi.org/project/cupy/ for list of prebuilt wheels by CUDA version.

A few notes on dependencies:
- `dask_cudf` should probably list `cupy, numpy, pandas, fsspec, pyarrow` as runtime requirements and not just get them transitively from the dependency on `cudf`. Those are listed in `setup.py` except for `pyarrow`, which should probably be added there as well.
- I also see dependencies on `toolz` (listed as `import tlz`) which seems to be a dependency of dask that we're getting transitively. https://github.com/dask/dask/issues/1849#issuecomment-806715040

This could all be handled in a separate PR, I just wanted to make some notes for the moment.